ui/pages/Project/components/CreateVariantButton.jsx

```diff
@@ -139,24 +148,21 @@ const validateHasTranscriptId = (value, allValues, props, name) => {
   return allValues[TRANSCRIPT_ID_FIELD_NAME] ? undefined : `Transcript ID is required to include ${name}`
 }
 
-const formatField = field => ({ inline: true, width: 16, ...field })
+const formatField = (field) => {
+  let formattedField = { inline: true, width: 16, ...field }
+  if (field.validate && field.validate !== validateHasTranscriptId) {
+    formattedField = { ...formattedField, label: `${field.label}*` }
+  }
+  return formattedField
+}
```

Comment:
you can use a `const` here because javascript objects are mutable. A cleaner way to do it might be
```suggestion
const formatField = (field) => {
  const formattedField = { inline: true, width: 16, ...field }
  if (field.validate && field.validate !== validateHasTranscriptId) {
    formattedField.label =`${field.label}*`
  }
  return formattedField
}
```